WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift

```diff
@@ -1897,13 +1897,31 @@ extension WooAnalyticsEvent {
 
         /// Tracked when the "learn more" button in the In-Person Payments onboarding is tapped.
         ///
-        /// - Parameter countryCode: the country code of the store.
+        /// - Parameters:
+        ///   - reason: the reason for viewing the learn more page.
+        ///   - countryCode: the country code of the store.
+        ///   - gatewayID: the plugin (e.g. "woocommerce-payments" or "woocommerce-gateway-stripe") to be included in the event properties in Tracks.
         ///
-        static func cardPresentOnboardingLearnMoreTapped(reason: String, countryCode: CountryCode) -> WooAnalyticsEvent {
+        static func cardPresentOnboardingLearnMoreTapped(reason: String,
+                                                         countryCode: CountryCode,
+                                                         gatewayID paymentGatewayID: String?) -> WooAnalyticsEvent {
             WooAnalyticsEvent(statName: .cardPresentOnboardingLearnMoreTapped,
                               properties: [
                                 Keys.countryCode: countryCode.rawValue,
-                                Keys.reason: reason
+                                Keys.reason: reason,
+                                Keys.gatewayID: gatewayID(forGatewayID: paymentGatewayID)
```

Comment:
maybe we can rename this to something like `safeGatewayID(forGatewayID gatewayID` or something like that?
Now it says gatewayId for gatewayId and it feels a bit weird :D

Reply:
Good suggestion! Refactored in https://github.com/woocommerce/woocommerce-ios/pull/11690/commits/14237ea1e157b403deb12d61741a277f8163ea7b